app/ide-desktop/lib/dashboard/src/App.tsx

```diff
@@ -255,19 +272,19 @@ function AppRouter(props: AppRouterProps) {
     }
   }, [/* should never change */ localStorage, /* should never change */ inputBindingsRaw])
 
-  React.useEffect(() => {
-    if (remoteBackend) {
-      void remoteBackend.logEvent('open_app')
-      const logCloseEvent = () => void remoteBackend.logEvent('close_app')
-      window.addEventListener('beforeunload', logCloseEvent)
-      return () => {
-        window.removeEventListener('beforeunload', logCloseEvent)
-        logCloseEvent()
-      }
-    } else {
-      return
-    }
-  }, [remoteBackend])
+  // React.useEffect(() => {
+  //   if (remoteBackend) {
+  //     void remoteBackend.logEvent('open_app')
```

Comment:
commented code, note that we want this to still be here though. perhaps this should be conditionally run only when not in dev mode?

Reply:
Yes, I remember, this is why this PR is in draft mode 😉